### PR TITLE
Guanlecoja bump

### PR DESCRIPTION
**Pull request by tardyp**
removed the doc fix from this, it is now part of the buildbot data PR, as I refactored the doc in that one.

Now travis is verifying guanlecoja against lot of version of node/npm

https://travis-ci.org/buildbot/guanlecoja

### CODE REVIEW

File: master/docs/developer/www.rst [hunk 1]
```diff
@@ -844,17 +844,24 @@ You will also need ``NodeJS``, and ``npm`` installed.
 Prerequisites
 ~~~~~~~~~~~~~
 
-* Install latest release of node.js.
+.. note::
 
-  http://nodejs.org/ is a good start for windows and osx.
+  Buildbot ui is only tested to build on node 4.x.x.
```
Comment (sa2ajj) [inline master/docs/developer/www.rst]: [nit] ui -> UI

File: master/docs/developer/www.rst [hunk 2]
```diff
@@ -844,17 +844,24 @@ You will also need ``NodeJS``, and ``npm`` installed.
...
+  There are known issues with node 5.x.x and especially npm 3.x.x. (:bug:`4496`).
 
-  For linux, as node.js is evolving very fast, distros versions are often too old. For ubuntu, for example, you want to use following ppa:
+* Install LTS release of node.js. 
+
+  http://nodejs.org/ is a good start for windows and osx
+
+  For linux, as node.js is evolving very fast, distros versions are often too old, and sometimes distro maintainers make incompatible changes (i.e naming node binary nodejs instead of node)
```
Comment (sa2ajj) [inline master/docs/developer/www.rst]: [nit] linux -> Linux

File: master/docs/developer/www.rst [hunk 3]
```diff
@@ -844,17 +844,24 @@ You will also need ``NodeJS``, and ``npm`` installed.
...
+  For ubuntu and other debian distros, you want to use following method:
```
Comment (sa2ajj) [inline master/docs/developer/www.rst]: [nit] ubuntu -> Ubuntu, debian -> Debian based distros

File: master/docs/developer/www.rst [hunk 4]
```diff
@@ -844,17 +844,24 @@ You will also need ``NodeJS``, and ``npm`` installed.
...
 
   .. code-block:: none
 
-    sudo add-apt-repository -y ppa:chris-lea/node.js
+    curl -sL https://deb.nodesource.com/setup_4.x | sudo bash -
 
   Please feel free to update this documentation for other distros.
+  Know good source for linux binary distribution is: https://github.com/nodesource/distributions
```
Comment (sa2ajj) [inline master/docs/developer/www.rst]: ditto

**Comment by tardyp** [comment]
@rutsky I created guanlecoja 0.5.2, which will normally work with npm 3

**Pull request merged**